lib/improver/tests/acceptance/test_blend_adjacent_points.py

```diff
@@ -92,9 +108,12 @@ def test_time(tmp_path):
     kgo_dir = acc.kgo_root() / "blend-adjacent-points/time_bounds"
     multi_prob = sorted(kgo_dir.glob("*wind_gust*.nc"))
     output_path = tmp_path / "output.nc"
-    args = ["time", "1536908400",
-            "--units", "seconds since 1970-01-01 00:00:00",
+    args = ["--coordinate", "time",
+            "--central-point", "1536908400",
+            "--units",
+            "seconds since 1970-01-01 00:00:00",
```

Comment:
These should be on the same line!  (Units are seconds since...)

Reply:
Yep..